src/api/handlers/guild.ts

```diff
@@ -462,20 +462,40 @@ export async function getMembers(
     throw new Error(Errors.MISSING_INTENT_GUILD_MEMBERS);
   }
 
-  const result = (await RequestManager.get(
-    endpoints.GUILD_MEMBERS(guildID),
-    options,
-  )) as MemberCreatePayload[];
+  const members = new Collection<string, Member>();
+  let loops = 1;
+  while (options?.limit ?? 1 > members.size) {
```

Comment:
This needs some modifications, pagination system should only run if the user provides NO limit as in if limit is undefined OR if the limit is more than 1000

Reply:
Can do that but actually not needed I made it so that the system only runs once if limit is under 1000 or undefined..

Reply:
The console.log should not occur in that case